src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -365,7 +366,8 @@ public static Delegate CreateDelegate(Type type, object target, string method, b
             {
                 if (throwOnBindFailure)
                     throw new ArgumentException(SR.Arg_DlgtTargMeth);
-                d = null;
+
+                return null;
```

Comment:
We could use the `!` operator here instead of using it at the call-sites. That way the return type wouldn't be nullable.

Reply:
Nevermind, I see it depends on the `throwOnBindfailure` argument to be set to null, disregard.

Reply:
> That way the return type wouldn't be nullable.

But the return type needs to be nullable.  This is a public API that can return null.

Reply:
Yep, I think I added the comment to disregard at the same time as you were typing 😆